go.mod

```diff
@@ -3,13 +3,20 @@ module github.com/nats-io/nats.go
 go 1.20
 
 require (
-	github.com/klauspost/compress v1.17.2
+	github.com/golang/protobuf v1.5.3
+	github.com/klauspost/compress v1.17.6
+	github.com/nats-io/nats-server/v2 v2.10.11
```

Comment:
I am not sure why, but this is required in order to compile, I ran `go mod tidy`

Reply:
you need to run with `go test -modfile=go_test.mod`